.github/CONTRIBUTING.md

```diff
@@ -58,13 +58,18 @@ First of all, thanks for your interest in contributing!
     git checkout -b name-of-your-branch
     ```
   4. Make change to your local copy of the folium repository
-  5. Commit those changes
+  5. Make sure the tests pass:
+    * in the repository folder do `pip install -e .`  (needed for notebook tests)
+    * along with all the dependencies install `phantomjs` via `npm install -g phantomjs`
```

Comment:
I don't have npm, I think I got it from their site. Maybe either say the npm part as hint, or tell people how to get npm.

Reply:
see new commits